src/utils.tsx

```diff
@@ -413,6 +414,77 @@ export const getInitialColumnFilterFnState = <TData extends MRT_RowData>(
   return initialState;
 };
 
+interface FilterFn {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (row: MRT_RowData, id: string, filterValue: any): boolean;
+}
+
+export const customFilterFunctions: Record<string, FilterFn> = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  arrIncludesNone: (row: MRT_RowData, id: string, filterValue: any) => {
+    if (Array.isArray(filterValue)) {
+      return !filterValue.includes(row.getValue(id));
+    }
+    return row.getValue(id) !== filterValue;
+  },
+};
+
+export const MRT_Functions_Localisation: Record<string, string> = {
+  filterArrIncludesNone: 'Excludes',
+};
+
+export type PropertyFiltersType = {
```

Comment:
Now that this has been moved to utils, the name PropertyFiltersType could be confusing. It might help to choose a more descriptive name that reflects its purpose and scope within utils.

Reply:
Not sure what a good name would be, but the new one of `ItemPropertyFiltersTypes` feels more confusing to me. It applies to items and catalogue items which is why Item wasn't mentioned in the name before. If its used in `ColumnFiltersType` perhaps `PropetyColumnFiltersType` is more consistent?

Reply:
It's only used in the context of item properties in columns, so I think calling it `PropertyColumnFiltersType` is better and works with its use case. Yeah, I forgot that it was used in both Catalogue Items and Items table, so I agree omitting `Item` is probably best.

I think maybe a name like `PropertyFilterVariantsType` and `ColumnFilterVariantsType` would be better? Makes it clearer that it's for setting filter variants, and not something like filter functions.

Reply:
Is this even needed anymore ??

Reply:
Do you mean using ColumnFiltersType everywhere instead of the more specific PropertyColumnFiltersType? You could replace it in the propertyFilters definition, you would just have to add fields for date, and datetime in the instantiation.

```
const propertyFilters: ColumnFiltersType = {
      boolean: 'autocomplete',
      string: 'text',
      number: 'range',
      null: 'text',
      date: 'date',
      datetime: 'datetime',
    };
```

Reply:
The `propertyFilters` for the item table is only used here 

```
export type ColumnFiltersType = PropertyFiltersType & {
  datetime: 'datetime-range' | 'datetime';
  date: 'date-range' | 'date';
};
```
it can be moved into this.

Reply:
Oh, yes I see what you mean. Yeah, it's not really being used individually in the items Table. The only reason I've kept it is because it's still used in Catalogue Items table, but once I've reworked that table, it'll no longer be needed.